src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -164,50 +167,53 @@ def test_triton_llm_profile_data(self, mock_read_write) -> None:
         assert metrics.num_output_tokens == [3, 6]
         assert metrics.num_input_tokens == [3, 4]
 
-        assert stat.avg_time_to_first_token == 2
-        assert stat.avg_inter_token_latency == 2
-        assert stat.avg_output_token_throughput_per_request == pytest.approx(
+        # Disable Pylance warnings for dynamically set attributes due to Statistics
+        # not having strict attributes listed.
+        assert stat.avg_time_to_first_token == 2  # type: ignore
+        assert stat.avg_inter_token_latency == 2  # type: ignore
+        assert stat.avg_output_token_throughput_per_request == pytest.approx(  # type: ignore
             np.mean(ottpr)
         )
-        assert stat.avg_num_output_token == 4.5
-        assert stat.avg_num_input_token == 3.5
+        assert stat.avg_num_output_token == 4.5  # type: ignore
```

Comment:
Should we be ignoring the entire file?

Reply:
No, I would rather not. It'd be better to improve the class at some point to be explicit with these, potentitally. But then we'd just miss all errors in this test.

Reply:
makes sense to me.
Interested to hear what improvements you envision. 
Still learning this alongside you.